.gitlab/pipeline/zombienet/bridges.yml

```diff
@@ -0,0 +1,58 @@
+# This file is part of .gitlab-ci.yml
+# Here are all jobs that are executed during "zombienet" stage for bridges
+
+# common settings for all zombienet jobs
+.zombienet-bridges-common:
+  before_script:
+    # Exit if the job is not merge queue
+    # - if [[ $CI_COMMIT_REF_NAME != *"gh-readonly-queue"* ]]; then echo "I will run only in a merge queue"; exit 0; fi
+    - echo "Zombienet Tests Config"
+    - echo "${ZOMBIENET_IMAGE}"
+    - echo "${GH_DIR}"
+    - echo "${LOCAL_DIR}"
+    - ls "${LOCAL_DIR}"
+    - export DEBUG=zombie,zombie::network-node
+    - export ZOMBIENET_INTEGRATION_TEST_IMAGE="${BRIDGES_ZOMBIENET_TESTS_IMAGE}":${BRIDGES_ZOMBIENET_TESTS_IMAGE_TAG}
+    - echo "${ZOMBIENET_INTEGRATION_TEST_IMAGE}"
+  stage: zombienet
+  image: "${BRIDGES_ZOMBIENET_TESTS_IMAGE}:${BRIDGES_ZOMBIENET_TESTS_IMAGE_TAG}"
+  needs:
+    - job: build-push-image-bridges-zombienet-tests
+      artifacts: true
+  extends:
+    - .kubernetes-env
+    - .zombienet-refs
+  variables:
+    BRIDGES_ZOMBIENET_TESTS_IMAGE_TAG: ${DOCKER_IMAGES_VERSION}
+    BRIDGES_ZOMBIENET_TESTS_IMAGE: "docker.io/paritypr/bridges-zombienet-tests"
+    GH_DIR: "https://github.com/paritytech/polkadot-sdk/tree/${CI_COMMIT_SHA}/bridges/zombienet"
+    LOCAL_DIR: "/builds/parity/mirrors/polkadot-sdk/bridges/zombienet"
+    FF_DISABLE_UMASK_FOR_DOCKER_EXECUTOR: 1
+    RUN_IN_CONTAINER: "1"
+  artifacts:
+    name: "${CI_JOB_NAME}_${CI_COMMIT_REF_NAME}_zombienet_bridge_tests"
+    when: always
+    expire_in: 2 days
+    paths:
+      - ./zombienet-logs
+  after_script:
+    - mkdir -p ./zombienet-logs
+    # copy logs of tests runner (run-tests.sh)
+    - cp -r /tmp/bridges-zombienet-tests.*/tmp.*/tmp.* ./zombienet-logs/
```

Comment:
Nit: Would be nice if this logs could have a more suggestive name

Reply:
Yeah, that's a good idea. But now there are just three log files matching this criteria, so not a big deal imo. When I/we will be adding more tests (hope to do that soon) it'll make sense to give meaningful names to those log files. Let's do it later in a separate PR